src/solve.jl

```diff
@@ -128,6 +128,13 @@ function solve(system::AbstractSystem,
         transformation_matrix = nothing
     end
 
+    # ==============================
+    # Sparse/dense matrix conversion
+    # ==============================
+    try
+        Δ = options[:assume_sparse] ? sparse(S.A) : full(S.A)
+    end
```

Comment:
isn't a `catch .. Δ = S.A` missing? i'm not sure what happens if the given `AbstractMatrix` doesn't implement `full` or `sparse`? we should check what happens with the `SparseMatrixExp` case.

Reply:
`try error("") end` works.

Reply:
`Δ` is already defined - it would just keep its value.

Reply:
Oh my, what did I do here? This was complete nonsense. Not it is fixed.